hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.test.js

```diff
@@ -1,54 +1,52 @@
 import React from "react";
 import { render } from "@testing-library/react";
-import ItemTable from "./ItemTable";
+import {
+    PendingTable,
+    ReturnedTable,
+    CheckedOutTable,
+} from "components/dashboard/ItemTable/ItemTable";
 import { itemsP, itemsR, itemsC } from "pages/Dashboard/Dashboard";
 
 it("renders correctly when there's an order pending and orange chip", () => {
```

Comment:
Wrap each of these in a `describe` block to indicate which component is being tested. Makes the file a little bit cleaner, and jest outputs will be grouped by component that way rather than showing every test if they all pass. For example:

```javascript
describe("<PendingTable />", () => {
    it("renders correctly when there's an order pending and orange chip", () => {
        ....
    });

    it(..., () => {
        ...
    });
});

describe("<ItemTable />", () => {
    ...
});
```